### DISCUSSION TITLE
I have a problem configuring the plugin to use British English

### DISCUSSION THREAD

**Discussion by duncanatt** [znck/grammarly#410] [General]
Hi,

I have set the following `"grammarly.config.documentDialect":"british"` in my VSCode `settings.json` (User) but to no avail. I still keep getting suggestions to replace words from British English to their American counterpart.

I have also double-checked that in my Grammarly account, the British dialect is configured Language Settings section.

Would appreciate any help.

**Comment by adrianbradu** [comment]
Yep, same here. It looks like a glitch in need of fixing 😃

**Comment by Denbergvanthijs** [comment]
I have the same problem!

**Comment by MirkMart** [comment]
Hi!
I managed to make it work both adding "grammarly.config.documentDialect":"british" to my settings.json and changing the dialect in the grammarly configuration of VSC extensions. With only the second pracaution it was not working until I precised it in the *.json

**Reply by Mont0111** [reply]
Doesn't changing the dialect in extension settings automatically change the .json file? I tried a few times changing the extension from auto-detect to British and the .json changes itself, but the extension is still correcting my spellings.

**Reply by MirkMart** [reply]
You are right. I have a json file at the base of my project folder in which I wrote the specification once more. Actually, I have to retrace my steps because it worked for a couple of hours and yesterday night it gave the error again
